mission_control/navigator_missions/nav_missions_lib/timeout_manager.py

```diff
@@ -5,12 +5,13 @@ class TimeoutManager(object):
 
     @classmethod
     def generate_timeouts(cls, time_left, missions_left):
```

Comment:
Should pass a real time left here, and set the last missions timeout with this value instead of 1000000